pkg/router/router.go

```diff
@@ -77,13 +77,17 @@ func registerKubeAPIProxy(r endpoints.Resource, container *restful.Container) {
 	proxy.Consumes(restful.MIME_JSON, "text/plain").
 		Produces(restful.MIME_JSON, "text/plain").
 		Path("/proxy")
+	proxy.Consumes(restful.MIME_JSON, "application/json-patch+json").
```

Comment:
can `"application/json-patch+json"` be added to the list above instead?
is it really needed for the `Produces` call too? Which APIs produce Patch JSON as output?

Reply:
We have added this to the list above, however we definitely need to keep both the consumes and produces parts as without these the patching does not work.

Reply:
<img width="1435" alt="Screenshot 2019-09-16 at 14 18 57" src="https://user-images.githubusercontent.com/32342288/64962046-9f3c5e00-d88e-11e9-932f-05d6ac6302e6.png">

Reply:
Looks like this change may have been reverted, `proxy.Consumes` still appears twice